src/borg/testsuite/archiver/diff_cmd.py

```diff
@@ -72,22 +72,20 @@ def test_basic_functionality(self):
         self.cmd(f"--repo={self.repository_location}", "create", "test1b", "input", "--chunker-params", "16,18,17,4095")
 
         def do_asserts(output, can_compare_ids, content_only=False):
-            # File contents changed (deleted and replaced with a new file)
-            change = "B" if can_compare_ids else "{:<19}".format("modified")
-            lines = output.splitlines()
+            lines: list = output.splitlines()
             assert "file_replaced" in output  # added to debug #3494
-            self.assert_line_exists(lines, f"{change}.*input/file_replaced")
+            self.assert_line_exists(lines, "modified.*input/file_replaced")
```

Comment:
you removed the specific testing for the can_compare_ids True case:
it used to test for "B...filename" (B being the unit [bytes] of the actual size of the detected changes).

for the can_compare_ids False case, it can not compute the actual size of the changes, because the ids are computed differently and so it can't detect what's identical and what not.